goformation_test.go

```diff
@@ -249,13 +249,6 @@ var _ = Describe("Goformation", func() {
 			Expect(template).ShouldNot(BeNil())
 		})
 
-		resources := template.GetAllCustomResources()
-
-		It("should have exactly one resource", func() {
-			Expect(resources).To(HaveLen(1))
-			Expect(resources).To(HaveKey("MyCustomResource"))
-		})
-
```

Comment:
What's the reason for removing this test?

Reply:
2 reasons 

1. I removed `GetAllCustomResources()` function seems that it do not make bigger sense to have it - then implementation of a resource is not necessarily different that others 
2. In the next lines (269-271) we can find 

resources, ok := result["Resources"].(map[string]interface{})
Expect(ok).To(BeTrue())
Expect(resources).To(HaveLen(1))
Expect(resources).To(HaveKey("MyCustomResource"))
			
Which is in my opinion almost the same test but without - template.GetAllCustomResources() - this is why I think we can remove that part of the test